ProcessMaker/Traits/MakeHttpRequests.php

```diff
@@ -303,7 +303,7 @@ private function response($response, array $data = [], array $config = [])
                 $content = json_decode($bodyContent, true);
                 break;
             case $status > 200 && $status < 300:
-                $content = [];
+                $content = empty($bodyContent) ? json_decode($bodyContent, true) : [];
```

Comment:
The goal is do not clear the content if the status is > 201, right? If that is the case, the condition of this line should be reversed:     $content = empty($bodyContent) ? [] : json_decode($bodyContent, true);

Additionally, this line should be present in the method "responseWithHeaderData" too

Reply:
Right, fixed